core/app/models/spree/promotion_handler/shipping.rb

```diff
@@ -1,7 +1,7 @@
 module Spree
   module PromotionHandler
     # Used for activating promotions with shipping rules
-    class FreeShipping
+    class Shipping
```

Comment:
We probably want to keep `Spree::PromotionHandler::FreeShipping` as a deprecated constant.

This is probably a good case for [ActiveSupport::Deprecation::DeprecatedConstantProxy](http://api.rubyonrails.org/classes/ActiveSupport/Deprecation/DeprecatedConstantProxy.html). I believe we re-add a `promotion_handler/free_shipping.rb` with

```
module Spree
  module PromotionHandler
    ActiveSupport::Deprecation::DeprecatedConstantProxy.new('FreeShipping', 'Shipping', Spree::Deprecation)
  end
end
```